src/current/_includes/releases/v23.1/v23.1.26.md

```diff
@@ -0,0 +1,39 @@
+## v23.1.26
+
+Release Date: September 12, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+<h3 id="v23-1-26-security-updates">Security updates</h3>
+
+- For clusters using [Cluster SSO using JSON Web Tokens (JWTs)]({% link v23.1/sso-sql.md %}), the [cluster setting]({% link v23.1/cluster-settings.md %}) `server.jwt_authentication.issuers` can now take multiple values to support various kinds of providers and their mapped JWKS URI. You can set it to one of the following values, which are parsed by Go:
+  1. A string that contains a single valid issuer URI. Example: `https://accounts.google.com`
+  1. A string that contains a JSON array of valid issuer URIs. Example: `['example.com/adfs','https://accounts.google.com']`
+  1. A string that contains a JSON map of valid issuer URIs to corresponding JWKS URIs. and deserialized into a map of issuer URLs to corresponding JWKS URIs. A JSON map overrides the JWKS URI published in the issuer's `well-known/` endpoint. Example: `'{    "issuer_jwks_map": {        "https://accounts.google.com": "https://www.googleapis.com/oauth2/v3/certs",        "example.com/adfs": "https://example.com/adfs/discovery/keys"     }    }'`
+
+  When `issuer_jwks_map` is set, the key set is fetched from the JWKS URI directly. Otherwise, when `JWKSAutoFetchEnabled` is set, the JWKS URI is fetched from the issuer's `well-known/` endpoint. [#128669][#128669]
+
+<h3 id="v23-1-26-operational-changes">Operational changes</h3>
+
+- New [structured logging events]({% link v23.1/logging.md %}) in the `OPS` channel report broken connections and related transactions during node shutdown.
+  - `node_shutdown_connection_timeout`: Logged if there are still open client connections after the timeout defined by `server.shutdown.connections.timeout` expires.
+  - `node_shutdown_transaction_timeout`: Logged if there are still open transactions on those open client connections after the timeout defined by `server.shutdown.transactions.timeout` expires. [#128709][#128709]
+
+<h3 id="v23-1-26-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug introduced in v23.1 in which output of [`EXPLAIN (OPT, REDACT)`]({% link v23.1/explain.md %}) of the following `CREATE` statements was not redacted:
+  -`EXPLAIN (OPT, REDACT) CREATE TABLE`
+  `EXPLAIN (OPT, REDACT) CREATE VIEW`
+  `EXPLAIN (OPT, REDACT) CREATE FUNCTION`
+
```

Comment:
There's a render problem with the remaining notes. I don't know if those sub-bullets in the suggestion above will fix it, so you may want to double check that locally.